axes/attempts.py

```diff
@@ -99,6 +80,55 @@ def reset_user_attempts(request: HttpRequest, credentials: dict = None) -> int:
     """
 
     attempts = filter_user_attempts(request, credentials)
+
+    count, _ = attempts.delete()
+    log.info('AXES: Reset %s access attempts from database.', count)
+
+    return count
+
+
+def reset(ip: str = None, username: str = None) -> int:
+    """
+    Reset records that match IP or username, and return the count of removed attempts.
+
+    This utility method is meant to be used from the CLI or via Python API.
+    """
+
+    attempts = AccessAttempt.objects.all()
+
+    if ip:
+        attempts = attempts.filter(ip_address=ip)
+    if username:
+        attempts = attempts.filter(username=username)
```

Comment:
@aleksihakli correct me if I'm wrong, but in case both `ip` and `username` are present, this will perform an `and` and not and `or` operation. So maybe in the docs would be worthy to mention this.

Reply:
Good note, I'll upgrade the docs accordingly.